src/realm.js

```diff
@@ -445,6 +447,15 @@ export class Realm {
     return !!this.createdObjectsTrackedForLeaks;
   }
 
+  evaluateWithoutLeakLogic(f: () => Value): Value {
+    this.ignoreLeakLogic = true;
```

Comment:
We should probably assert this isn't already true here. Since otherwise you'd get really confusing results if these ever end up nested.